unlock-app/public/templates/airdrop.csv

```diff
@@ -0,0 +1,2 @@
+recipient,count,email,expiration,expire,manager
+julien51.eth,1,julien@unlock-protocol.com,0,true,julien51.eth
```

Comment:
hehe :D

Reply:
I assume `0` for expiration means `never` ? We should probably document in the guide on how to airdrop memberships.

Reply:
0 would be default. I think that's the convention we use in extend and other APIs on unlockjs. 

for never expiring, there is `expire` which is more explicit since we cannot assume lack of expiration as user not meaning to expire something.

Reply:
Hum, ok if no `expire` let's also use the lock's duration (as for 0).

Reply:
if no expire and expiration provided, we use the default yes.